src/html-to-govspeak.js

```diff
@@ -85,8 +85,14 @@ service.addRule('heading', {
     const number = node.nodeName.charAt(1)
     if (number === '1' || number === '2') {
       prefix = '## '
-    } else if (number === '3' || number === '4' || number === '5') {
+    } else if (number === '3') {
       prefix = '### '
+    } else if (number === '4') {
+      prefix = '#### '
+    } else if (number === '5') {
+      prefix = '##### '
+    } else if (number === '6') {
+      prefix = '###### '
     } else {
       prefix = ''
     }
```

Comment:
I wonder if we could DRY up this implementation a bit? 🤔 It feels a bit repetitive.

My initial thoughts were something like this:

```js
let number = node.nodeName.charAt(1)
if (number == "1") number = "2"
const prefix = "#".repeat(number)

return `\n\n${prefix} ${content}\n\n`
```

### Browser support

The only potential issue is that [String.prototype.repeat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat) isn't supported by IE11.

I'm assuming this package needs to support IE11?

It looks like other non-IE11 compatible features are in use here – e.g. [ES6 Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) – but I also note that Babel is in use, which I believe [will convert them](https://babeljs.io/docs/en/babel-plugin-transform-template-literals) if IE11 is one of the target browsers. But I don't think it polyfills `String.prototype.repeat()`.